espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/SetOrientation.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.handlers
+
+import java.util.Arrays
+
+import androidx.test.espresso.ViewInteraction
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
+import io.appium.espressoserver.lib.model.Element
+import io.appium.espressoserver.lib.model.OrientationParams
+import io.appium.espressoserver.lib.viewaction.OrientationChange
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+
+class SetOrientation : RequestHandler<OrientationParams, Void?> {
+
+    @Throws(AppiumException::class)
+    override fun handle(params: OrientationParams): Void? {
+        params.orientation?.let {orientation ->
```

Comment:
Throw the exception here if orientation is null (I think I've already seen that in the other Pr)